mri/test/test_wavelet_adjoint.py

```diff
@@ -80,6 +85,40 @@ def test_Wavelet3D_PyWt(self):
             numpy.testing.assert_allclose(x_d, x_ad, rtol=1e-6)
         print(" Wavelet3 adjoint test passes")
 
+    def test_Wavelet_UD_2D(self):
+        """Test the adjoint operation for Undecimated wavelet
+        """
+        for i in range(self.max_iter):
+            print("Process Wavelet Undecimated test '{0}'...", i)
+            wavelet_op = WaveletUD2(nb_scale=4)
+            Img = (numpy.random.randn(self.N, self.N) +
```

Comment:
nitpicking here, but generally in Python the variables respect snake_case, so this should be `img` (and `i_p`).
More generally, the following variable names are a bit unclear.

Reply:
Coding style, while I agree with you, this is how it is currently for all the tests. 
Maybe we fix this in #36 ? 
For now, I feel either we fix all or fix none.

Reply:
Had an offline discussion, To ensure we have clean check-ins, I am convinced that we make this change now. Will update in next commit